app/experimenter/templates/experiments/detail_base.html

```diff
@@ -231,9 +231,29 @@ <h5 class="modal-title">Edited by {{user}} </h5>
                             <tbody>
                                 {% for key, value in user_change.changed_values.items %}
                               <tr>
+                                {% if key == "variants" %}
                                 <th scope="row">{{key}}</th>
-                                  <td>{{value.old_value}}</td>
-                                  <td>{{value.new_value}}</td>
+                                <td>
+                                  {% for variant in value.old_value %}
+                                    <strong>Variant: </strong>
+                                      {% for var_key, var_value in variant.items %}
+                                        <p class="ml-3"> {{var_key}}: {{var_value}} </p>
+                                      {% endfor %}
+                                  {% endfor %}
+                                </td>
+                                <td>
+                                    {% for variant in value.new_value %}
+                                      <strong>Variant: </strong>
+                                        {% for var_key, var_value in variant.items %}
+                                          <p class="ml-3"> {{var_key}}: {{var_value}} </p>
```

Comment:
Wrap the key in `<strong>`, take out the colon, and put the value on a separate line. Example:

**name**
Feature Enabled